src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -929,6 +970,44 @@ else if (FreeMode.SUPPLYCAMP == Settings.instance.getFreeMode())
         }
     }
 
+    private String blockListToCommaSeparatedString(List<BlockPos> blocksToReportList) {
+        StringBuilder outputListStringBuilder = new StringBuilder();
+        boolean firstItem = true;
+        for (BlockPos blockPos : blocksToReportList) {
+            if (firstItem) {
+                firstItem = false;
+            } else {
+                outputListStringBuilder.append(',');
+                outputListStringBuilder.append(' ');
+            }
+            outputListStringBuilder.append('(');
+            outputListStringBuilder.append(blockPos.getX());
+            outputListStringBuilder.append(' ');
+            outputListStringBuilder.append(blockPos.getY());
+            outputListStringBuilder.append(' ');
+            outputListStringBuilder.append(blockPos.getZ());
+            outputListStringBuilder.append(')');
+        }
+        String outputList = outputListStringBuilder.toString();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "outputList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1488)